test/integration/request_tracing_test.rb

```diff
@@ -2,12 +2,18 @@
 
 class RequestTracingTest < ActionDispatch::IntegrationTest
   setup do
+    Sidekiq::Testing.inline!
+
     @govuk_request_id = "12345-67890"
     @draft_edition = create(:draft_publication)
     @presenter = PublishingApiPresenters.presenter_for(@draft_edition)
     login_as(create(:gds_admin))
   end
 
+  teardown do
+    Sidekiq::Testing.fake!
```

Comment:
Minor: It might be more robust to use the construct I used in [this commit][1] here, i.e. to restore the Sidekiq test mode to what it was before it was changed to inline. 

[1]: https://github.com/alphagov/whitehall/commit/0eada72c6c29d123df316d96808f77a60d7232cf

Reply:
I've adopted your approach and rewritten the commit. Thanks!